### PR TITLE
Feat/icon-redux-action

### PR DESCRIPTION
**Pull request by damien5701**

### CODE REVIEW

File: example/ios/Podfile.lock [hunk 1]
```diff
@@ -236,8 +236,10 @@ PODS:
     - React-cxxreact (= 0.63.4)
     - React-jsi (= 0.63.4)
   - React-jsinspector (0.63.4)
-  - react-native-netinfo (6.0.0):
-    - React-Core
+  - react-native-config (0.12.0):
```
Comment (imranMnts) [inline example/ios/Podfile.lock]: react-native-config

File: README.md [hunk 2]
```diff
@@ -218,4 +258,5 @@ sessionManager.request(...)
 | interceptIOS | Boolean  |   false   |     no      | Intercept native iOS requests                                |
 | disableShake | Boolean  |   false   |     no      | Set to true to disable shake feature to display Netwatch     |
 | maxRequests  |  Number  |    100    |     no      | Maximum requests displayed                                   |
+| reduxConfig  |  Object  |    {}     |     no      | Extra infos for Redux Action                                 |
```
Comment (imranMnts) [inline README.md]: default value is currently undefined

 src/index.tsx l.200

File: example/ios/Podfile.lock (hunk 1 above)
Comment (damien5701) [reply example/ios/Podfile.lock]: Cleaning done

File: README.md (hunk 2 above)
Comment (damien5701) [reply README.md]: Ok, at last, I added a default value corresponding to the readme. Also iconMapper that receive this props is defined to {} by default. (see ReduxLogger.tsx l.8) and I prefer avoid undefined value.

**Review by imranMnts — APPROVED** [review]

**Pull request merged**